Mobile/lib/Components/ColorScheme.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'dart:math';
+
+class CustomColorScheme {
+  static final random = Random();
+  
+  static List<Color> colorScheme = const [
+    Color.fromRGBO(194, 232, 255, 77),
```

Comment:
Hvorfor er den blå farve mere gennemsigtig end de andre?
Jeg vil sige at de skulle alle være ens (ingen alpha)
```suggestion
    Color.fromRGBO(194, 232, 255, 1),
```